docs/2.8.0/docs/canton/usermanual/docker.rst

```diff
@@ -8,15 +8,21 @@
 Running in Docker
 =================
 
+.. note::
+
+	Please note that the images have moved from ``digitalasset-canton-enterprise-docker.jfrog.io/digitalasset/canton-enterprise``
+        to ``digitalasset-docker.jfrog.io/canton-enterprise`` as per version 2.8.0. The old images will remain, but new versions won't 
+        be added to the old repository. In addition, a docker image with the open source version is no longer provided.
```

Comment:
```suggestion
        be added to the old repository. In addition, a Docker image with the open source version is no longer provided.
```
For clarity, would this be better as "a Docker image is no longer provided with the open source version of Canton"?

Reply:
Another option: The open source Canton Docker image is no longer available.

(I'm seeing it this way: the Docker image *is/was* one form of the open source version... not something provided "with" it)

Reply:
Also wanted to note that we have a separate work item of improving Docker instructions everywhere, so we will likely expand/revise this later (we can go with an "MVP" edition for now) 🙂

Reply:
fixed